docs/source/posts/2022/2023-01-24-final-report-praneeth.rst

```diff
@@ -0,0 +1,226 @@
+.. image:: https://developers.google.com/open-source/gsoc/resources/downloads/GSoC-logo-horizontal.svg
+   :height: 50
+   :align: center
+   :target: https://summerofcode.withgoogle.com/programs/2022/projects/a47CQL2Z
+
+.. image:: https://www.python.org/static/community_logos/python-logo.png
+   :width: 40%
+   :target: https://summerofcode.withgoogle.com/programs/2022/organizations/python-software-foundation
+
+.. image:: https://python-gsoc.org/logos/FURY.png
+   :width: 25%
+   :target: https://fury.gl/latest/index.html
+
+Google Summer of Code Final Work Product
+========================================
+
+.. post:: January 24 2023
+   :author: Praneeth Shetty
+   :tags: google
+   :category: gsoc
+
+-  **Name:** Praneeth Shetty
+-  **Organisation:** Python Software Foundation
+-  **Sub-Organisation:** FURY
+-  **Project:** `FURY - Improve UI elements for drawing geometrical
+   shapes <https://github.com/fury-gl/fury/wiki/Google-Summer-of-Code-2022-(GSOC2022)#project-5-improve-ui-elements-for-drawing-geometrical-shapes>`_
+
+
+Proposed Objectives
+-------------------
+
+-  Visualization UI:
+
+   -  Drawing Geometrical Objects
+   -  Moving Components
+   -  Rotating Components
+   -  Erasing Components
+   -  Resizing Components
+
+-  Stretch Goals:
+
+   -  Converting 2D shapes to 3D
+
+Objectives Completed
+--------------------
+
+
+-  **DrawPanel (previously known as Visualization UI)**
+
+   ``DrawPanel`` is the parent component that contains and manages all the other sub-components to efficiently visualize the shapes. The main functions of the ``DrawPanel`` are capturing interactions from user, managing modes, drawing shapes and transforming them. The ``DrawPanel`` is mainly divided into three parts :
+
+   i. **Main Panel**
+        It is the main background panel(``Panel2D``) on which the main interaction and visualization happen. Here user can interactively draw shapes, reposition and rotate them. This panel also defines the boundaries for the shapes. It can also be called as a container element as it containes all the shapes and other DrawPanel components.
+   ii. **Mode Panel** 
+        It is a composite UI element consisting of the main panel(``Panel2D``) on which buttons(``Button2D``) are arranged which can toggle the current working mode. Each button has an icon associated with it which tries to depict the information about the mode. Here mode is nothing but the different channels which on selection can perform different tasks. Some of the modes present in the Mode Panel are discussed below:
+            -  Selection:  This mode is used to select an individual or group of shapes.
+            -  Deletion:  This mode is used to delete an individual or group of shapes.
+            -  The modes mentioned below create an element on the Panel which is described below.
+                -  Line: 
+                -  Quad: 
+                -  Circle: 
+                -  Polyline: 
+                -  Freehand drawing: 
```

Comment:
why there are `:` at the end of each line?

Reply:
In my gist, I added the shape icons for each but as this is a blog the temporary icons may not be available in future so removed it but forgot to remove the colons.
I have updated the blog.